diff --git a/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerModule.kt b/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerModule.kt
index fbb6836..e75b6b2 100644
--- a/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerModule.kt
+++ b/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerModule.kt
@@ -51,7 +51,7 @@ class DocumentScannerModule(reactContext: ReactApplicationContext) :
 
     val documentScannerOptionsBuilder = GmsDocumentScannerOptions.Builder()
       .setResultFormats(GmsDocumentScannerOptions.RESULT_FORMAT_JPEG)
-      .setScannerMode(GmsDocumentScannerOptions.SCANNER_MODE_FULL)
+      .setScannerMode(GmsDocumentScannerOptions.SCANNER_MODE_BASE)
 
     if (options.hasKey("maxNumDocuments")) {
       documentScannerOptionsBuilder.setPageLimit(
diff --git a/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerPackage.kt b/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerPackage.kt
index 84f3fec..54a8f53 100644
--- a/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerPackage.kt
+++ b/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerPackage.kt
@@ -1,33 +1,16 @@
 package com.documentscanner
 
-import com.facebook.react.BaseReactPackage
+import com.facebook.react.ReactPackage
 import com.facebook.react.bridge.NativeModule
 import com.facebook.react.bridge.ReactApplicationContext
-import com.facebook.react.module.model.ReactModuleInfo
-import com.facebook.react.module.model.ReactModuleInfoProvider
-import java.util.HashMap
+import com.facebook.react.uimanager.ViewManager
 
-class DocumentScannerPackage : BaseReactPackage() {
-  override fun getModule(name: String, reactContext: ReactApplicationContext): NativeModule? {
-    return if (name == DocumentScannerModule.NAME) {
-      DocumentScannerModule(reactContext)
-    } else {
-      null
-    }
+class DocumentScannerPackage : ReactPackage {
+  override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
+    return listOf(DocumentScannerModule(reactContext))
   }
 
-  override fun getReactModuleInfoProvider(): ReactModuleInfoProvider {
-    return ReactModuleInfoProvider {
-      val moduleInfos: MutableMap<String, ReactModuleInfo> = HashMap()
-      moduleInfos[DocumentScannerModule.NAME] = ReactModuleInfo(
-        DocumentScannerModule.NAME,
-        DocumentScannerModule.NAME,
-        false,  // canOverrideExistingModule
-        false,  // needsEagerInit
-        false,  // isCxxModule
-        true // isTurboModule
-      )
-      moduleInfos
-    }
+  override fun createViewManagers(reactContext: ReactApplicationContext): List<ViewManager<*, *>> {
+    return emptyList()
   }
 }
